third_party/move/move-prover/move-docgen/src/docgen.rs

```diff
@@ -1788,12 +1789,17 @@ impl<'env> Docgen<'env> {
                     // consume the remaining '`'. Report an error if we find an unmatched '`'.
                     assert_eq!(chars.next(), Some('`'), "Missing backtick found in {} while generating documentation for the following text: \"{}\"", self.current_module.as_ref().unwrap().get_name().display_full(self.env), text);
 
-                    write!(
-                        &mut decorated_text,
-                        "<code>{}</code>",
-                        self.decorate_code(&code)
-                    )
-                    .unwrap()
+                    if self.options.is_mdx_compatible() {
+                        // Don't decorate code here if it's MDX
+                        write!(&mut decorated_text, "`{}`", code).unwrap()
```

Comment:
Decoration of code inserts links to all known symbols in the code. For example, if there is the name of a struct or a function, there will be a link to its definition in the docs. If you do not do it here, this feature is lost, which is quite a pity. Why did you skip this?

That is BTW the reason why we use `<code>` below because you cannot embed links into backslashes.

Reply:
I skipped it because the code blocks don't properly work outside of GitHub.  Which is why you use raw HTML.

I know it's a pity, but this is a path to getting some docs up and running for the doc site for now.  To me having markdown compatible docs (as an optional choice), is a win over the links.